llama_stack/cli/model/describe.py

```diff
@@ -7,8 +7,6 @@
 import argparse
 import json
 
-from termcolor import colored
```

Comment:
Why is this gone?

Reply:
`colored` is used for model before, and the header has `style` by default, so no need to use `colored`, please correct me if I am understanding.